ci-automation/image.sh

```diff
@@ -98,7 +95,50 @@ function _image_build_impl() {
                           --output_root="${CONTAINER_IMAGE_ROOT}" \
                           --torcx_root="${CONTAINER_TORCX_ROOT}" prodtar container
 
-    # rename container and push to build cache
-    docker_commit_to_buildcache "${image_container}" "${image}" "${docker_vernum}"
+    # copy resulting images + push to buildcache
+    local images_out="images/"
+    rm -rf "${images_out}"
+    ./run_sdk_container -n "${image_container}" -C "${packages_image}" \
+            -v "${vernum}" \
+            mv "${CONTAINER_IMAGE_ROOT}/${arch}-usr/" "./${images_out}/"
+    cd "images/latest"
+    sign_artifacts "${SIGNER}" *
+    copy_to_buildcache "images/${arch}/${vernum}/" *
+
+    (
+    set +x
+    # Don't fail the whole job
+    set +e
```

Comment:
Would this work (to limit the scope of a different working directory to the subshell)?

```suggestion
    sign_artifacts "${SIGNER}" "images/latest/"*
    copy_to_buildcache "images/${arch}/${vernum}/" "images/latest/"*

    (
    set +x
    # Don't fail the whole job
    set +e
    cd "images/latest"
```

Reply:
I've added two wip commits and started a new build to test if it really uploads the correct paths http://192.168.42.7:8080/job/container/job/packages/691/cldsv/

Reply:
Seems to work https://bincache.flatcar-linux.net/images/amd64/9999.1.2+noimagecontainer-kai2/

Reply:
(Had to deal with way too many git conflicts…)

Reply:
Thanks for doing it despite the conflicts. :)